### DISCUSSION TITLE
WebView2 :: InvalidOperationException

### DISCUSSION THREAD

**Discussion by sigmarsson** [microsoft/microsoft-ui-xaml#8114] [Q&A]
Why does it occur sometimes ? 

<img width="683" alt="image" src="https://user-images.githubusercontent.com/43340959/213511308-a480211d-934b-429d-b143-fe9ec9afd30d.png">

**Comment by codendone** [comment]
Do you see this error in the debug output:
```
WinRT information: NavigateToString(): Failed because a valid CoreWebView2 is not present. Make sure one was created, for example by calling EnsureCoreWebView2Async() API.
```
? This sounds like it could be #3714.

**Reply by sigmarsson** [reply]
Will tell you whenever again it will be thrown.
